pkgs/by-name/gi/giza/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  autoreconfHook,
+  pkg-config,
+  gfortran,
+  cairo,
+  freetype,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "giza";
+  version = "1.4.1";
+
+  src = fetchFromGitHub {
+    owner = "danieljprice";
+    repo = "giza";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-spb46IoySf6DM454adcWmqqLlzNA2HK9z29TzOCECJ4=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    pkg-config
+    gfortran
+  ];
+
+  buildInputs = [
+    cairo
+    freetype
+  ];
+
+  meta = {
+    description = "A scientific plotting library for C/Fortran";
```

```suggestion
    description = "Scientific plotting library for C/Fortran";
```

I haven't reviewed a package addition in a while, but it looks like the guidelines say to [not use articles](https://github.com/NixOS/nixpkgs/blob/0abfc619bcb605299a0f3f01c1887bb65db61a6b/pkgs/README.md?plain=1#L376).

This should really be automated though, so: https://github.com/NixOS/nixpkgs-vet/pull/104 :)

---

> looks like the guidelines say to [not use articles](https://github.com/NixOS/nixpkgs/blob/0abfc619bcb605299a0f3f01c1887bb65db61a6b/pkgs/README.md?plain=1#L376).

I learned something new today, thanks!